src/core/fonts.js

```diff
@@ -718,6 +718,10 @@ var Font = (function FontClosure() {
   function int16(b0, b1) {
     return (b0 << 8) + b1;
   }
+  
+  function neg16(value) {
+    return (value)?(value-0x10000):value;
```

Comment:
Nit: this line is missing spaces before/after all operators, it should be e.g. like this instead:
```javascript
return value ? value - 0x10000 : value;
```